### DISCUSSION TITLE
Getting 1.19

### DISCUSSION THREAD

**Discussion by ghost** [Minestom/Minestom#1355] [Help] [answered]
Hello I was searching for a 1.19 version of minestom, but could only see 1.19.1 and 1.18.2

**Comment by ryleu** [comment, marked as answer]
There is no 1.19 version of Minestom. You can use ViaVersion on the Velocity proxy with Minestom 1.18, or you can wait for Minestom 1.19.2 and then disable whatever features you don’t like.

**Comment by ghost** [comment]
Oh okay thats sad well thanks for your quick answer!

**Reply by emortaldev** [reply]
I don't know why you'd want specifically 1.19 when there's little to no changes in .1 and .2 (and security fixes)

**Reply by ghost** [reply]
1.19.2 has the player report system and honestly 1.19.1 I don't know what they chnaged but I went for it now

**Reply by TheMode** [reply]
1.19.1 has the reporting feature. But you can work around it by sending system messages
